packages/storage/src/platform/node/connection.ts

```diff
@@ -21,7 +21,7 @@ import {
   ErrorCode
 } from '../../implementation/connection';
 import { internalError } from '../../implementation/error';
-import nodeFetch, { Headers } from 'node-fetch';
+import axios, { AxiosResponseHeaders, AxiosError } from 'axios';
```

Comment:
It looks like you're just using request, can you replace this with `import { AxiosResponseHeaders, AxiosError, request } from 'axios'`? Otherwise we'll be bringing in the entire package it seems.

Reply:
Disregard this. It looks like `request` is only available via default import. This is fine.